src/components/internals/rich-text-body/rich-text-body-button.js

```diff
@@ -1,59 +1,71 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
 import { css } from '@emotion/core';
-import AccessibleButton from '../../buttons/accessible-button';
 import vars from '../../../../materials/custom-properties';
 
+const propsToOmit = ['isActive', 'label', 'isDisabled'];
+
+function getFillColor(props) {
+  if (props.isActive) return vars.colorSurface;
+  return vars.colorSolid;
+}
+
 const RichTextBodyButton = props => {
-  const { onMouseDown } = props;
-  const handleMouseDown = React.useCallback(
-    event => {
-      event.preventDefault();
-      onMouseDown(event);
-    },
-    [onMouseDown]
-  );
+  const restOfProps = omit(props, propsToOmit);
+
   return (
-    <div>
-      <AccessibleButton
-        isDisabled={props.isDisabled}
```

Comment:
Since you're no longer using the `AccessibleButton`, should the `MouseDown` prop be removed from it?

Reply:
yep done